hack/run_test

```diff
@@ -338,6 +331,37 @@ function run_replication_test() {
   done
 }
 
+function setup_replication_cluster() {
+  # Run the PostgreSQL master
+  run_master $cid_suffix
+  local master_ip
+  master_ip=$(get_container_ip master-$cid_suffix.cid)
+
+  # Run the PostgreSQL slaves
+  local i
+  for i in $(seq ${SLAVE_NUM:-1}); do
+    run_slave $master_ip $cid_suffix-$i
+  done
+
+  # Check if the master knows about the slaves
+  CONTAINER_IP=$master_ip
+  test_slave_visibility $cid_suffix
+}
+
+function run_replication_test() {
+  echo "Testing master-slave replication"
+  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
+  local cid_suffix="basic"
+  local table_name="t1"
+  max_attempts=30
+
+  # Setup the cluster
+  setup_replication_cluster
+
+  # Do some real work to test replication in practice
+  test_value_replication $table_name
+}
+
```

Comment:
Please don't move the functions, if possible.  It is harder to review (from the diff is not obvious what changed within function).